src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerTest.java

```diff
@@ -565,13 +565,20 @@ public void given_initiation_task_with_followup_overdue_ctg_when_cancelled_with_
     public void given_event_requestHearingRequirementsFeature_when_initiated_verfiy_task_creation() {
         String caseId1 = UUID.randomUUID().toString();
         final String taskId = initiateTaskForGivenId(caseId1, "requestHearingRequirementsFeature",
```

Comment:
This is calling initiateTaskForGivenId with the wrong number of args. (The method has 5 args in the implementation below). Causes compilation error on CEH

Reply:
Fixed sorry something must have gone wrong on the rebase